specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/keys.json

```diff
@@ -1303,14 +1301,6 @@
                 "value": "oct-HSM",
                 "description": "Octet sequence (used to represent symmetric keys) which is stored the HSM."
               },
```

Comment:
You'll need to remove all terminating commas here and throughout. It's not valid JSON, and various validation and generation tools _will_ fail.

Reply:
Thanks Heath. Fixed commas as part of the next commit.